utils/coverage_files/coverage_percentage.txt

```diff
@@ -1 +1 @@
-39.39
+50.51
```

Comment:
is this updated automatically? or still needed?

Reply:
removing this file